### PR TITLE
Player

### PR DESCRIPTION
**Pull request by IStallcup**
Initial commit of player class!

### CODE REVIEW

File: src/main/java/models/Player.java [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+    public int score;
+
+    public void empty_hand()
+    {
+        int hand_size = hand.size();
+        for (int i = 0; i < hand_size; i++)
+            hand.remove(0);
+    }
+
+    public int computeScore()
```
Comment (athai) [inline src/main/java/models/Player.java]: Score is already calculated in a function below. :-P

Comment (IStallcup) [inline src/main/java/models/Player.java]: On it! :+1:

File: src/main/java/models/Player.java [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+    public int score;
+
+    public void empty_hand()
+    {
+        int hand_size = hand.size();
+        for (int i = 0; i < hand_size; i++)
+            hand.remove(0);
+    }
+
+    public int setCardVal(Card card){
```
Comment (athai) [inline src/main/java/models/Player.java:23]: Might one to add a test for this method later but other than that, looks good! +1

File: src/main/java/models/Player.java [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+public abstract class Player
+{
+    //"oops" not in the constructor
+    public java.util.List<Card> hand = new ArrayList<>();
+
+    public int score;
+
+    public void empty_hand()
+    {
+        int hand_size = hand.size();
```
Comment (kannabanana) [inline src/main/java/models/Player.java:18]: This should be in a while loop (until the size is 0) keep removing cards

**Pull request merged**